frontend/server/src/DAO/Users.php

```diff
@@ -93,7 +93,11 @@ public static function savePassword(\OmegaUp\DAO\VO\Users $Users): int {
         return \OmegaUp\MySQLConnection::getInstance()->Affected_Rows();
     }
 
-    final public static function getExtendedProfileDataByPk($user_id) {
+    /**
+     * @param int $user_id
+     * @return null|array{country: ?string, country_id: ?int, state: ?string, state_id: ?int, school: ?string, school_id: ?int, graduation_date: ?string, email: string, locale: ?string}
+     */
+    final public static function getExtendedProfileDataByPk(?int $user_id): ?array {
```

Comment:
```suggestion
    final public static function getExtendedProfileDataByPk(int $userId): array {
```

mejor hay que pedirle al que llama esta funci'on que valide que el ID no sea nulo.